.github/workflows/ci.yaml

```diff
@@ -95,7 +95,7 @@ jobs:
         with:
           path: ~/.cache/pip
           key: ${{ hashFiles('pyproject.toml') }}
-      - run: cat ci-test-connections.yaml > test-connections.yaml
+      - run: cat .github/ci-test-connections.yaml > test-connections.yaml
```

Comment:
Since we moved it to be inside the `.github` folder, we could also rename the file to be `test-connections.yaml`

Reply:
@kaxil I noticed the CI is breaking, it may be because of this change :thinking: 
I had a quick look at `conftest.py` and I couldn't spot why this is not working - @dimberman  could you give us a hand?